shipyard_blueprints/airtable/shipyard_airtable/airtable.py

```diff
@@ -28,57 +28,73 @@ def _handle_error(self, error: HTTPError) -> None:
         logger.debug(
```

Comment:
Throw this debug message in the CLI portion and remove it from here

Reply:
I would push a bit on this because it is necessary debugging information specific to the `_handle_error`. 

what I was referring to in slack was moving the message from the catch all exception into the cli portion but after thinking it through a bit more it would be useful in the class to avoid duplicating code between blueprints.